pkgs/development/python-modules/scikit-hep-testdata/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, fetchFromGitHub
+, python
+, buildPythonPackage
+, importlib-resources
+, pyyaml
+, requests
+, pytestCheckHook
+, setuptools_scm
```

The old identifier is still working but for new packages we should stick to the naming schema according [PEP 0503](https://nixos.org/manual/nixpkgs/stable/#contributing).

```suggestion
, setuptools-scm
```